configure

```diff
@@ -102,7 +102,7 @@ forcesse2=0
 avx2flag="-mavx2"
 sse2flag="-msse2"
 ssse3flag="-mssse3"
-sse4flag="-msse4"
+sse41flag="-msse4.1"
 sse42flag="-msse4.2"
```

Comment:
Need to update these two lines in configure script:
```
echo sse4flag = $sse4flag >> configure.log
```
We should also add one for sse42flag.

And then update this one:
```
/^SSE4FLAG *=/s#=.*#=$sse4flag#
```
And add one for SSE42.